package.json

```diff
@@ -9,7 +9,10 @@
     "debug": "open 'rndebugger://set-debugger-loc?host=localhost&port=8081'",
     "test": "jest",
     "lint": "eslint .",
-    "lint:fix": "eslint . --fix"
+    "lint:fix": "eslint . --fix",
+    "android-fingerprint": "keytool -list -v -keystore android/app/debug.keystore -alias androiddebugkey -storepass android -keypass android",
+    "presetup": "node ./scripts/setup.js",
+    "setup": "cd ios && pod install"
```

Comment:
Since we're having the `setup` script actually change the context of the directory, I would say we should have the script return to the root directory:
`"setup": "cd ios && pod install && cd .."`

Reply:
It looks like from experiment that the node script namespace is different from the actual executable namespace. So if I'm running the script from root, and the script does a `cd ios` and then exits, I'm still back in the root. So adding an extra `&& cd ..` was unnecessary.

Also right now the setup script does ios stuff, but I bet in the future we'll need it to do more. So don't think of the content of the script, rather the outcome of the script to determine the name. In this case, we also hid the `presetup` behind the setup which does updates to both android and iOS. We could also just change setup to be `cd ios && pod install && node ./scripts/setup.js`. Either way is fine, this way might be good in that it's an example of using `pre` node scripts... but it could also be bad in that people might not realize right away that `presetup` will run first when you run `setup`. What do you think?

Reply:
That makes sense. I didn't know that it doesn't actually change the context of the active directory once it finishes executing so that's good to know.

I think the scripts make more sense with the assumption that we will expand upon it in the future. I'm fine with the way it is then